src/js/node/http.ts

```diff
@@ -527,6 +523,7 @@ Server.prototype.listen = function (port, host, backlog, onListen) {
     this[serverSymbol] = Bun.serve<any>({
       tls,
       port,
+      reusePort: !cluster.isPrimary,
```

Comment:
This would break SO_REUSEPORT in node:http. It is an option the developer can pass to node:http. We should support that option.